src/Microsoft.AspNet.Mvc.TagHelpers/ScriptTagHelper.cs

```diff
@@ -197,17 +197,20 @@ public class ScriptTagHelper : TagHelper
             var attributes = new TagHelperAttributeList(output.Attributes);
 
             var builder = new DefaultTagHelperContent();
-            var originalContent = await context.GetChildContentAsync();
```

Comment:
@NTaylorMullen this change doesn't remove a capability. When using `PostElement`, there's no reason to read the original element's content.